iree/compiler/Dialect/VM/IR/VMOps.td

```diff
@@ -1372,6 +1372,27 @@ class VM_BinaryComparisonOp<Type type, string mnemonic, VM_OPC opcode,
   ];
 }
 
+class VM_BinaryComparisonPseudoOp<Type type, string mnemonic,
```

Comment:
Are these comparison ops covered (indirectly) by unit tests? I could be overlooking a sign swap in the op folders and bugs could be tricky to track down if starting at higher levels.

Reply:
good point - now that I have iree/vm/test/ I'll go add some there now (there are dedicated folder tests, but no dedicated runtime tests)